qcodes/instrument_drivers/rohde_schwarz/RTO1000.py

```diff
@@ -129,6 +129,146 @@ def get_raw(self):
+                        'SG2TL1',   'SG2TL2',
+                        'SG3TL1',   'SG3TL2',
+                        'SG4TL1',   'SG4TL2',
+                        'Z1V1',     'Z1V2',     'Z1V3',     'Z1V4',
+                        'Z1I1',     'Z1I2',     'Z1I3',     'Z1I4',
+                        'Z2V1',     'Z2V2',     'Z2V3',     'Z2V4',
+                        'Z2I1',     'Z2I2',     'Z2I3',     'Z2I4')
+
+        self.categories = vals.Enum('AMPTime',   'JITTer', 'EYEJitter', 'SPECtrum',
+                                    'HISTogram', 'PROTocol')
+
+        self.meas_type = vals.Enum(
+                        # Amplitude/time measurements
+                        'HIGH',        'LOW',          'AMPLitude',    'MAXimum',
+                        'MINimum',     'PDELta',       'MEAN',         'RMS',
+                        'STDDev',      'POVershoot',   'NOVershoot',   'AREA',
+                        'RTIMe',       'FTIMe',        'PPULse',       'NPULse',
+                        'PERiod',      'FREQuency',    'PDCYcle',      'NDCYcle',
+                        'CYCarea',     'CYCMean',      'CYCRms',       'CYCStddev',
+                        'PULCnt',      'DELay',        'PHASe',        'BWIDth',
+                        'PSWitching',  'NSWitching',   'PULSetrain',   'EDGecount',
+                        'SHT',         'SHR',          'DTOTrigger',   'PROBemeter',
+                        'SLERising',   'SLEFalling'
+                        # Jitter measurements
+                        'CCJitter',     'NCJitter',    'CCWidth',      'CCDutycycle',
+                        'TIE',          'UINTerval',   'DRATe',        'SKWDelay',
+                        'SKWPhase',
+                        # Eye diagram measurements
+                        'ERPercent',    'ERDB',         'EHEight',      'EWIDth',
+                        'ETOP',         'EBASe',        'QFACtor',      'RMSNoise',
+                        'SNRatio',      'DCDistortion', 'ERTime',       'EFTime',
+                        'EBRate',       'EAMPlitude',   'PPJitter',     'STDJitter',
+                        'RMSJitter',
+                        # Spectrum measurements
+                        'CPOWer',       'OBWidth',      'SBWidth',      'THD',
+                        'THDPCT',       'THDA',         'THDU',         'THDR',
+                        'HAR',          'PLISt',
+                        # Histogram measurements
+                        'WCOunt',       'WSAMples',     'HSAMples',     'HPEak',
+                        'PEAK',         'UPEakvalue',   'LPEakvalue',   'HMAXimum',
+                        'HMINimum',     'MEDian',       'MAXMin',       'HMEan',
+                        'HSTDdev',      'M1STddev',     'M2STddev',     'M3STddev',
+                        'MKPositive',   'MKNegative'
+                        )
+
+        self.add_parameter('enable',
+                           label='Measurement {} enable'.format(meas_nr),
```

Comment:
Do you know you can do this now: 
```python
label=f'Measurement {meas_nr} enable'
```

Reply:
I did not know this. It's a nice feature! Updated the driver to this new format.